x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/trial_license_complete_tier/index.ts

```diff
@@ -8,7 +8,7 @@
 import { FtrProviderContext } from '../../../../ftr_provider_context';
 
 export default function ({ loadTestFile }: FtrProviderContext) {
-  describe('@ess @serverless SecuritySolution Saved Objects', () => {
+  describe('@ess @serverless @serverlessQA SecuritySolution Saved Objects', () => {
```

Comment:
@MadameSheema Is it alright to add the @serverlessQA tag for the api tests?

@logeekal If @MadameSheema has confirmed that we want to add the @serverlessQA tags in this PR, please apply them for the explore serverless tests as well. Thank you!

Reply:
Hello @logeekal , after checking with @MadameSheema , the @serverlessQA tags should not be applied in this PR.

Reply:
As we discussed already, on this PR we should not be adding the `@serverlessQA` tag since we need to make sure first that the tests are stable.

In order to do so, we need to have the tests executed as part of the periodic pipeline.

Reply:
Thanks @MadameSheema , I will remove it.